### PR TITLE
Update test and readme

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
+`wolff` will queue pending messages on disk so it can survive from message loss
+in case of application, network or kafka disturbances.
+
+In case of producer restart, messages queued on disk are replayed towards kafka,
+however, async callback functions are not evaluated upon acknowledgements received
+from kafka for replayed messages.
+
+## More flexible connection management
+
+`wolff` provids `per_partition` and `per_broker` connection management strategy.
```
Comment (id) [inline README.md:19]: provides

File: README.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+In case of `per_partition` strategy, `wolff` establishes one TCP connection for
+to partition leader. `brod` however, only establishes connections per-broker, that
```
Comment (id) [inline README.md:21]: "for to partition leader"? as in "per partition leader"?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+is if two partition leaders happen to reside on the same broker, they will have
+to share the same TCP connection.
+
+Therer is still a lack of benchmarking to tell the difference of how performant
```
Comment (id) [inline README.md:25]: There

**Comment by zmstone** [comment]
thanks for the review @id updated with a force push

**Review by id — APPROVED** [review]

**Pull request merged**